geoserver/webapp/pom.xml

```diff
@@ -169,42 +169,6 @@
       <artifactId>gt-image</artifactId>
       <version>${gt.version}</version>
     </dependency>
-    <!-- These are not specified in the web-app module upstream pom.xml
-      but are implicit -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-beans</artifactId>
-      <version>${gs.spring.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-core</artifactId>
-      <version>${gs.spring.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-context</artifactId>
-      <version>${gs.spring.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-webmvc</artifactId>
-      <version>${gs.spring.version}</version>
-    </dependency>
-    <!-- gs-sec-ldap explicitely needs v4.2.5, but the GS webapp upstream
-         uses v4.3.7 -->
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-tx</artifactId>
-      <version>${gs.spring.version}</version>
-    </dependency>
-    <!-- xml-apis is needed by gt-svg in v1.3.04, but the GS webapp upstream
-         uses v1.4.01 -->
-    <dependency>
-      <groupId>xml-apis</groupId>
-      <artifactId>xml-apis</artifactId>
-      <version>1.4.01</version>
-    </dependency>
```

Comment:
Why removing the dependencies ? Are they now implicit by using the spring boot parent pom ?